### PR TITLE
Merge upstream

### LINKED ISSUE #393: Helm Chart: Icon is missing

### PR DESCRIPTION
**Pull request by richard-cox**
Contains fix for .. Helm Chart: Icon is missing …
- fixes #393

### CODE REVIEW

File: deploy/kubernetes/console/templates/__stratos.tpl [hunk 1]
```diff
@@ -2,4 +2,14 @@
 
 # Extra env vars for the Jetstream Pod in deployment.yaml
 {{- define "stratosJetstreamEnv" }}
+        - name: MONOCULAR_CRT_PATH
```
Comment (richard-cox) [inline deploy/kubernetes/console/templates/__stratos.tpl]: Remove all indentation

File: README.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # Stratos
 
-<a href="https://travis-ci.org/cloudfoundry/stratos/branches"><img src="https://travis-ci.org/cloudfoundry/stratos.svg?branch=master"></a>&nbsp;<a style="padding-left: 4px" href="https://codeclimate.com/github/cloudfoundry/stratos/maintainability"><img src="https://api.codeclimate.com/v1/badges/61af8b605f385e894632/maintainability" /></a>
+<a href="https://travis-ci.com/cloudfoundry/stratos/branches"><img src="https://travis-ci.com/cloudfoundry/stratos.svg?branch=master"></a>&nbsp;<a style="padding-left: 4px" href="https://codeclimate.com/github/cloudfoundry/stratos/maintainability"><img src="https://api.codeclimate.com/v1/badges/61af8b605f385e894632/maintainability" /></a>
```
Comment (richard-cox) [inline README.md]: suse/stratos

**Review by richard-cox — COMMENTED** [review]
See comments

**Review by nwmac — APPROVED** [review]
LGTM

**Pull request merged**